CHANGELOG.md

```diff
@@ -10,16 +10,17 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ## [NEXT RELEASE]
 
 ### Added Features
+- A new `cluster` flag has been added to the `roxctl` commands and APIs that perform image scans, this enables delegating scans to specific secured clusters on demand.
+- Ad-hoc image scans delegated to secured clusters will now attempt to infer a namespace from the image path, this enables namespace secrets to be used for registry authentication which in turn enables scanning images from the OCP integrated registry.
```

Comment:
Can you prefix with ticket here, too?

Can you make it clear this is just for OCP internal registry? I think this can be made clearer by rephrasing the sentence to put the OCP integrated registry part earlier. Maybe like this:

```
Ad-hoc image scanning is now enabled for images in the OCP integrated registry.
  - StackRox attempts to infer the OCP project from the image path and utilize the namespace (project?) secrets for registry authentication.
```

Reply:
It's is based on the OCP integrated registry convention, but is not 'technically' limited to the OCP integrated registry.

Reply:
https://github.com/stackrox/stackrox/pull/7931/commits/b660b8282db3ff72b01afd20cc0e1932e1532a98